ReactAndroid/src/main/java/com/facebook/react/views/switchview/ReactSwitchManager.java

```diff
@@ -117,6 +122,18 @@ public void setOn(ReactSwitch view, boolean on) {
     view.setOnCheckedChangeListener(ON_CHECKED_CHANGE_LISTENER);
   }
 
+  @ReactProp(name = "thumbTintColor", customType = "Color")
+  public void setThumbTintColor(ReactSwitch view, Integer color) {
+    int thumbTintColor = color != null ? color : DEFAULT_THUMB_TINT_COLOR;
+    view.getThumbDrawable().setColorFilter(thumbTintColor, PorterDuff.Mode.MULTIPLY);
```

Comment:
Does passing setColorFilter(null) works to reset to default instead of having to provide default colors?